### PR TITLE
docs: update the template list in the docs

### PR DESCRIPTION
**Pull request by hassan254-prog**
## Describe your changes

- Update the template list in the docs

## Issue ticket number and link
[EXT-127](https://linear.app/nango/issue/EXT-127/update-the-template-list-in-the-docs)

### CODE REVIEW

File: docs-v2/integrations/integration-templates/bamboohr-basic.mdx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: 'Bamboohr Basic API Integration Template'
+sidebarTitle: 'Bamboohr Basic'
```
Comment (bodinsamuel) [inline docs-v2/integrations/integration-templates/bamboohr-basic.mdx:3]: I know it's like that in providers but I still don't know: is it Bamboohr basic auth or "basic" plan or something like that?

File: docs-v2/integrations/integration-templates/google-mail.mdx [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
     Learn how to use integration templates in Nango
 </Card>
 
-<Card title="Get the Gmail template"
+<Card title="Get the Google Mail template"
       href="https://github.com/NangoHQ/nango/tree/master/integration-templates/google-mail"
       icon="github">
-    Get the latest version of the Gmail integration template from GitHub
+    Get the latest version of the Google Mail integration template from GitHub
 </Card>
```
Comment (bodinsamuel) [inline docs-v2/integrations/integration-templates/google-mail.mdx:18]: @bastienbeurier Should we add a button to point to the UI? It's a pity we have a way to test templates in one click and it's not advertised

**Review by bodinsamuel — APPROVED** [review]
🧹💨

File: docs-v2/integrations/integration-templates/google-mail.mdx (hunk 2 above)
Comment (bastienbeurier) [reply docs-v2/integrations/integration-templates/google-mail.mdx:18]: Do you envision a button to link from the docs to enable the template in the app?

**Review by bastienbeurier — APPROVED** [review]

File: docs-v2/integrations/integration-templates/google-mail.mdx (hunk 2 above)
Comment (bodinsamuel) [reply docs-v2/integrations/integration-templates/google-mail.mdx:18]: We can think about it, there is no way to point to a template without having enabled an integration but that would be nice imo

File: docs-v2/integrations/integration-templates/bamboohr-basic.mdx (hunk 1 above)
Comment (hassan254-prog) [reply docs-v2/integrations/integration-templates/bamboohr-basic.mdx:3]: It is the Bamboohr basic auth.

Comment (bodinsamuel) [reply docs-v2/integrations/integration-templates/bamboohr-basic.mdx:3]: naming could be improved in the future :p

**Pull request merged**